script/tool/lib/src/lint_android_command.dart

```diff
@@ -96,4 +101,36 @@ gradle.projectsEvaluated {
 
     return failed ? PackageResult.fail() : PackageResult.success();
   }
+
+  /// Returns whether the plugin project is configured to enable all Gradle
+  /// lints and treat them as errors.
+  bool _mainGradleHasLintConfig(RepositoryPackage package) {
+    final List<String> gradleBuildContents = package
+        .platformDirectory(FlutterPlatform.android)
+        .childFile('build.gradle')
+        .readAsLinesSync();
+    return gradleBuildContents
```

Comment:
Consider writing this match to ignore lines that are commented out.

Reply:
I can make that change if you want; there are potential downsides though. The current approach is to enforce that we haven't just forgotten to set this up when adding a new package. Allowing commenting it out allows for someone to explicitly say "I know that this is supposed to be set up, but I'm turning it off for now for a specific reason". I actually did exactly that with the `javac` part of this file previously, where for packages that weren't ready yet I added the line it was looking for commented out, with a TODO referencing an issue.

That may not be a useful thing to support for this though, where we can use the baseline file as an escape hatch.